drivers/usb/device/usb_dc_stm32.c

```diff
@@ -636,10 +636,18 @@ int usb_dc_ep_configure(const struct usb_dc_ep_cfg_data * const ep_cfg)
 		    (usb_dc_stm32_state.pma_offset + ep_cfg->ep_mps)) {
 			return -EINVAL;
 		}
```

Comment:
I guess this check should take into account the ISOCHRONOUS case as well.

Reply:
yes

Reply:
Remains to be fixed